test/features/personalization/personalization.test.js

```diff
@@ -377,6 +377,13 @@ describe('Functional Test', () => {
     expect(document.querySelector('meta[property="og:title"]').content).to.equal('New Title');
     expect(document.querySelector('meta[property="og:image"]').content).to.equal('https://adobe.com/path/to/image.jpg');
   });
+
+  it('will add id to the section div', async () => {
+    addSectionAnchors(document);
+    const sectionWithId = document.querySelector('#marquee-container');
+    expect(sectionWithId).to.exist;
+    expect(sectionWithId.getAttribute('id')).to.equal('marquee-container');
```

Comment:
Nit: since we're querying for the id already, is there any situation where this could fail?